io-engine/src/core/env.rs

```diff
@@ -183,7 +183,7 @@ pub struct MayastorCliArgs {
     )]
     /// Persistent store timeout.
     pub ps_timeout: Duration,
-    #[clap(long = "ps-retries", default_value = "30")]
+    #[clap(long = "ps-retries", default_value = "255")]
```

Comment:
Modifying this might affect runtime of existing test.
Can leave this as is and modify  through the helm chart instead?

Reply:
Need to add a new value in that case. Can do that.

Reply:
https://github.com/openebs/mayastor-extensions/pull/592